src/components/views/close-contact-alert.tsx

```diff
@@ -40,7 +40,7 @@ export const CloseContactAlert: FC = () => {
           new Date(Number(contacts[0].exposureAlertDate)),
           contacts[0].daysSinceLastExposure
         );
-        setCloseContactDate(format(exposureDate, 'MMMM dd, yyyy'));
+        setCloseContactDate(format(exposureDate, 'MMMM do, yyyy'));
```

Comment:
This is the wrong format. `do` will display `2nd` rather than `2`

Reply:
That is what is in the design

https://app.zeplin.io/project/5f23ee3d1eb2f9215942942b/screen/5f354870e975187eabbe6cfe

Reply:
I cannot see that link. Nonetheless, I do not think we are intending to show the day of the week here (which starts with `0th`). I think you want `Do`

Reply:
https://date-fns.org/docs/format says `do` is day of month, `Do` is day of year...

<img width="1089" alt="Screenshot 2020-09-03 at 16 40 20" src="https://user-images.githubusercontent.com/1176557/92136646-29fdcc00-ee04-11ea-9ac0-50fb2467d8c1.png">

Reply:
Interesting. I was pretty sure the docs I looked at said something else. Nonetheless, I discussed this with the UX team. We do not want to include the `st` or `nd` stuff on the day of the month. It should be `September 1, 2020`

Reply:
Incidentally, this https://date-fns.org/v1.28.5/docs/format reflects what I was saying about the format string. I guess they changed it in some version.

Reply:
@dharding Those are version 1.x docs, here are version 2.x docs (which we're using) https://date-fns.org/v2.16.1/docs/format

Reply:
Also I was thinking to be most natural we'd do dates like `January 2, 2020` when written and `January 2nd, 2020` when spoken - do you think `January 2, 2020` is more natural when spoken in text-to-voice?

Reply:
@dharding @helloantoine can the designs be updated to ensure what we are delivering is what people are looking at...

Reply:
https://nearform.invisionapp.com/d/main#/console/20287854/428104629/preview